src/norm/postgres.nim

```diff
@@ -123,9 +123,9 @@ proc createTables*[T: Model](dbConn; obj: T) =
           const selfTableName = '"' & T.getCustomPragmaVal(tableName) & '"'
         else:
           const selfTableName = '"' & $T & '"'
-        fkGroups.add "FOREIGN KEY ($#) REFERENCES $#(id)" % [fld, selfTableName]
+        fkGroups.add """FOREIGN KEY ("$#") REFERENCES $#(id)""" % [fld, selfTableName]
       else:
-        fkGroups.add "FOREIGN KEY ($#) REFERENCES $#(id)" % [fld, (obj.dot(fld).getCustomPragmaVal(fk)).table]
+        fkGroups.add """FOREIGN KEY ("$#") REFERENCES $#(id)""" % [fld, (obj.dot(fld).getCustomPragmaVal(fk)).table]
```

Comment:
I had to add quotation marks here because otherwise the FK constraints wouldn't have any.
Please note that I ONLY added the quotation marks in these two lines.
There is a third line dealing with FK constraints, line 107 further up, where I DID NOT add quotation marks around the field,  because when that line gets called by tests the field suddenly already has quotationmarks (possibly because it calls the col proc where the others don't).

Reply:
> when that line gets called by tests the field suddenly already has quotationmarks (possibly because it calls the col proc where the others don't).

Oof, that's dangerous behavior. col proc should be called always.

Reply:
Just tried it, it does work.
I managed to replace `fld` on there with `obj.col(fld)` and it ran through all tests.
I'm not sure if the code that generates is correct, as stated I haven't read through this code in any intensity in order to understand what's going on, but it does pass the tests.